client/js/components/user/DpCreateItem.vue

```diff
@@ -195,8 +188,9 @@ export default {
     },
 
     itemResource () {
+      const type = this.entity === 'user' ? 'AdministratableUser' : this.entity
```

Comment:
I don't know how urgent this PR is, but can't we use `'AdministratableUser'` as the value for `this.entity` instead of `'user'`? Also maybe it makes sense to rename `entity` to `resourceType`? And shouldn't the resourceType for organisation be `'Organisation'`, too? :thinking:

Reply:
The PR already is way larger then supposed to be. And in my experience its not simple to rename a ResourceType or anything along. I thought about it too and quickly became the feeling of "one not simply.... "